pubnub-kotlin/pubnub-kotlin-impl/src/integrationTest/kotlin/com/pubnub/api/integration/HistoryIntegrationTest.kt

```diff
@@ -176,7 +176,7 @@ class HistoryIntegrationTest : BaseIntegrationTest() {
                                 expectedActionValue to
                                     listOf(
                                         PNFetchMessageItem.Action(
-                                            actionTimetoken = actionResult.actionTimetoken.toString(),
+                                            actionTimetoken = actionResult.actionTimetoken!!,
```

Comment:
Do we have integration test that checks if `PatchValue` works as expected?
I am thinking about full scenario: 
createMetadata, 
updateMetadata, 
validate some value has been set to null
validate some value has been remove
validate some value has been added

Then when different part of functionality(sending to server) will be done such test could be just lightly modified.
What do you think?

Reply:
Yes, I modified existing integration tests in two places to make sure that the data is read correctly from the server. I also have a unit test that checks that deserialization works correctly just for reading PatchValue from JSON.

But you're right that maybe once we have sending to the server, we should check the whole "journey" of the data to server and back explicitly. Will add a todo